src/chartiq_injections/backingStore.js

```diff
@@ -16,4 +16,21 @@ export const BackingStore = () => {
         this.adjustedDisplayPixelRatio = 1;
         this.draw();
     };
+
+    CIQ.ChartEngine.prototype.reconstituteBackingStore = function () {
+        if (!this.useBackingStore || !this.backing) return;
```

Comment:
@kseniya-deriv just to make a more clear, it's better to put a note here that we changed this line.